bboxes_labelling/combine_and_format_annotations.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+
+from annotations_utils import combine_multiple_via_jsons, convert_via_json_to_coco
+
+
+def main(
+    parent_dir_via_jsons,
+    via_default_dir,
+    via_project_name,
+):
+    # Get list of VIA JSON files
+    list_json_files = [
+        x
+        for x in Path(parent_dir_via_jsons).glob("*")
+        if x.is_file() and str(x).endswith(".json")
+    ]
+    list_json_files.sort()
+
+    # Combine VIA JSONS
+    json_out_fullpath = combine_multiple_via_jsons(
+        list_json_files,
+        via_default_dir=via_default_dir,
+        via_project_name=via_project_name,
+    )
+
+    # Convert to COCO
+    coco_out_fullpath = convert_via_json_to_coco(json_out_fullpath)
+
+    return coco_out_fullpath
+
+
+if __name__ == "__main__":
+    parent_dir_via_jsons = (
```

Comment:
I know the latest annotations are here, but maybe not hardcoded here will be better? We can have it as an args in bash script. What do you think?

Reply:
yes, 100%. I was doubting whether to add this to the PR, but I figured this way it's at least under version control...

If it's alright I would say we merge this for now, and open an issue on refactoring / restructuring the whole codebase, with an explicit reference to this?